### PR TITLE
Update fstg01 and fstg02 after `g_forest` refactor

### LINKED ISSUE #715: Updates corresponding to refactor of `g_forest`
Update implementation of `g_forest` following https://github.com/insightsengineering/tern/pull/1158

Merge after https://github.com/insightsengineering/tern/pull/1158 and once chevron release is complete.

### PR DESCRIPTION
**Pull request by edelarua**
Closes #715 

Merge after chevron release and https://github.com/insightsengineering/tern/pull/1158

### CODE REVIEW

File: R/fstg02.R
Comment (clarkliming) [inline R/fstg02.R:78]:
```suggestion
    ...,
    font_size = 7
```

to allow font_size be updated through `...`

File: DESCRIPTION
Comment (clarkliming) [inline DESCRIPTION:43]:
```suggestion
    tern (>= 0.9.3.9001),
```

**Comment by clarkliming** [comment]
thank you @edelarua . Let's wait until chevron released to CRAN and then merge this

**Comment by BFalquet** [comment]
test is failing because of a glitch in rlistings, see https://github.com/insightsengineering/rlistings/issues/195

**Review by BFalquet — APPROVED** [review]
I am approving it because the remaining issue has to be fixed in `rlistings`, not here.

**Pull request merged**
